src/executables/util_service.c

```diff
@@ -3977,6 +3981,9 @@ us_hb_deactivate (const char *hostname, bool immediate_stop)
       args[opt_idx++] = COMMDB_HB_DEACT_IMMEDIATELY;
     }
 
+  /* stop javasp server */
+  (void) process_javasp (STOP, 0, NULL, false, true, false, false);
```

Comment:
Is it right that the last arg is false ?
I think that the arg should be true for checking HA server.